common/artifact-manager/src/test/java/com/vmware/pscoe/iac/artifact/store/vrang/VraNgCustomFormTest.java

```diff
@@ -0,0 +1,174 @@
+        + "                   ]\n"
+        + "                }\n"
+        + "             ],\n"
+        + "             \"state\":{\n"
+        + "                \n"
+        + "             }\n"
+        + "          }\n"
+        + "       ]\n"
+        + "    },\n"
+        + "    \"schema\":{\n"
+        + "       \"project\":{\n"
+        + "          \"label\":\"Project\",\n"
+        + "          \"type\":{\n"
+        + "             \"dataType\":\"string\",\n"
+        + "             \"isMultiple\":false\n"
+        + "          },\n"
+        + "          \"valueList\":{\n"
+        + "             \"id\":\"projects\",\n"
+        + "             \"type\":\"scriptAction\"\n"
+        + "          },\n"
+        + "          \"constraints\":{\n"
+        + "             \"required\":true\n"
+        + "          }\n"
+        + "       },\n"
+        + "       \"numberOne\":{\n"
+        + "          \"label\":\"Number One\",\n"
+        + "          \"type\":{\n"
+        + "             \"dataType\":\"decimal\",\n"
+        + "             \"isMultiple\":false\n"
+        + "          },\n"
+        + "          \"default\":1,\n"
+        + "          \"constraints\":{\n"
+        + "             \"required\":true\n"
+        + "          }\n"
+        + "       },\n"
+        + "       \"numberTwo\":{\n"
+        + "          \"label\":\"Number Two\",\n"
+        + "         \"type\":{\n"
+        + "             \"dataType\":\"decimal\",\n"
+        + "             \"isMultiple\":false\n"
+        + "          },\n"
+        + "          \"default\":2,\n"
+        + "          \"constraints\":{\n"
+        + "             \"required\":true\n"
+        + "          }\n"
+        + "       },\n"
+        + "       \"deploymentName\":{\n"
+        + "          \"label\":\"Deployment Name\",\n"
+        + "          \"type\":{\n"
+        + "             \"dataType\":\"string\",\n"
+        + "             \"isMultiple\":false\n"
+        + "          },\n"
+        + "          \"constraints\":{\n"
+        + "             \"required\":true,\n"
+        + "             \"max-value\":80\n"
+        + "          }\n"
+        + "       }\n"
+        + "    },\n"
+        + "    \"options\":{\n"
+        + "       \"externalValidations\":[\n"
+        + "          \n"
+        + "       ]\n"
+        + "    }\n"
+        + " }\n";
+
+        // WHEN
+        VraNgCustomForm restFormSource = new VraNgCustomForm("123",
+            "Some Name", jsonString, null,
+            "com.vmw.vro.workflow", "requestForm",
+            "requestForm", "ON", "JSON");
+        VraNgCustomFormAndData repoFormSource =
+        new VraNgCustomFormAndData("123",
+            "Some Name", null, null,
+            "com.vmw.vro.workflow", "requestForm",
+            "requestForm", "ON", "JSON");
+
+        repoFormSource.setForm(jsonString);
+        JsonElement form = repoFormSource.getForm();
+
+        new VraNgCustomForm(repoFormSource);
```

Comment:
I think this constructor here is not needed (as it is not used anywhere).

Reply:
As this is a test, my idea was to invoke the constructor in order to verify (assert) that it does not throw an exception. 

Changing it to 
```
assertDoesNotThrow(() -> new VraNgCustomForm(repoFormSource));
```
and moving it into the `//THEN` section of the test for more clarity.